extensions/amp-auto-ads/amp-auto-ads-spec.md

```diff
@@ -0,0 +1,268 @@
+      position.
+    </td>
+  </tr>
+</table>
+
+#### AnchorObj
+<table>
+  <tr>
+    <th>Field Name</th>
+    <th>Required</th>
+    <th>Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>selector</td>
+    <td>Yes</td>
+    <td>string</td>
+    <td></td>
+    <td>
+      A CSS selector to select the element(s) at this level of the anchor
+      definition.
+    </td>
+  </tr>
+  <tr>
+    <td>index</td>
+    <td>No</td>
+    <td>number</td>
+    <td>0 (if <code>all</code> field not set to true)</td>
+    <td>
+      The index of the elements selected by the selector that this level of the
+      anchor definition should be limited to.
+    </td>
+  </tr>
+  <tr>
+    <td>all</td>
+    <td>No</td>
+    <td>boolean</td>
+    <td>false</td>
+    <td>
+      Ignored if index field set. If true then indicates that all elements
+      selected by the selector should be included.
+    </td>
+  </tr>
+  <tr>
+    <td>min_c</td>
+    <td>No</td>
+    <td>number</td>
+    <td>0</td>
+    <td>
+      The minimum length of an element's textContent property for it to be
+      included.
+    </td>
+  </tr>
+  <tr>
+    <td>sub</td>
+    <td>No</td>
+    <td>AnchorObj</td>
+    <td></td>
+    <td>
+      A recursive <code>AnchorObj</code> that will select elements within any elements
+      selected at this level of anchor definition.
+    </td>
+  </tr>
+</table>
+
+#### PlacementStyleObj
+<table>
+  <tr>
+    <th>Field Name</th>
+    <th>Required</th>
+    <th>Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>top_m</td>
+    <td>No</td>
+    <td>number</td>
+    <td>0</td>
+    <td>
+      The top margin in pixels that an ad inserted in this position should have.
+    </td>
+  </tr>
+  <tr>
+    <td>bot_m</td>
+    <td>No</td>
+    <td>number</td>
+    <td>0</td>
+    <td>
+      The bottom margin in pixels that an ad inserted in this position should
+      have.
+    </td>
+  </tr>
+</table>
+
+#### RelativePositionEnum
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>BEFORE</td>
+    <td>1</td>
+    <td>Ad should be inserted as sibling immediately before the anchor.</td>
+  </tr>
+  <tr>
+    <td>FIRST_CHILD</td>
+    <td>2</td>
+    <td>Ad should be inserted as the first child of the anchor.</td>
+  </tr>
+  <tr>
+    <td>LAST_CHILD</td>
+    <td>3</td>
+    <td>Ad should be inserted as the last child of the anchor.</td>
+  </tr>
+  <tr>
+    <td>AFTER</td>
+    <td>4</td>
+    <td>Ad should be inserted as sibling immediately after the anchor.</td>
+  </tr>
+</table>
+
+#### PlacementTypeEnum
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>BANNER</td>
+    <td>1</td>
+    <td>Placement describes a banner ad position.</td>
+  </tr>
+</table>
+
+### Example Configuration
+
+Specifies positions immediately after all `<P class='paragraph'>` elements that
```

Comment:
Rewrite the starting of the sentence as follows:

The following example specifies that the ad should be positioned immediately ...